### DISCUSSION TITLE
Example for odrl:constraint needed

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tractusx-edc#559] [Q&A] [answered]
Provided by @ds-mwesener 

Hello, does somebody has a link to an example of how the odrl:constraint object in the PolicyDefinitionRequest looks like? Swagger API / Postman Collection do not show an example. (EDC 0.4.1)

**Comment by stephanbcbauer** [comment]
provided by @wolf4ood On the latest postman collection there is an example how to create a policy with a constraint
https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/development/postman

**Comment by stephanbcbauer** [comment, marked as answer]
This is just copy & paste from the postmancolllection. Does it help? @ds-mwesener 

```json
{
    "@context": {
        "odrl": "http://www.w3.org/ns/odrl/2/"
    },
    "@type": "PolicyDefinitionRequestDto",
    "@id": "{{POLICY_ID}}",
    "policy": {
		"@type": "Policy",
		"odrl:permission" : [{
			"odrl:action" : "USE",
			"odrl:constraint" : {
				"@type": "LogicalConstraint",
				"odrl:or" : [{
					"@type" : "Constraint",
					"odrl:leftOperand" : "BusinessPartnerNumber",
					"odrl:operator" : {
                        "@id": "odrl:eq"
                    },
					"odrl:rightOperand" : "{{POLICY_BPN}}"
				}]
			}
		}]
    }
}
```

**Reply by ds-mwesener** [reply]
Thanks that helped.
